docker/Dockerfile.cpu

```diff
@@ -34,7 +34,6 @@ RUN git clone --recursive --branch $MMDET_BRANCH https://github.com/opencv/mmdet
 WORKDIR /mmdetection
```

Comment:
Could you also change `MMDET_BRANCH` a few lines above?

Reply:
done